src/checker/mod.rs

```diff
@@ -98,6 +101,19 @@ where
         }
     }
 
+    #[cfg(feature = "nlprules")]
+    {
+        // if config.is_enabled() {
```

Comment:
nlprules doesn't need a onfig, so you can't check that here right?

Reply:
Right now the associated type for config is `()` tbd

Reply:
Since I know that there is the necessity to cache the language rules in a dir, there will be a directory for caching those.

Reply:
Correction: License change for the binary with possible overrides.